packages/base-map/src/index.tsx

```diff
@@ -151,9 +151,9 @@ const BaseMap = ({
         <Styled.LayerSelector
           className="filter-group"
           id="filter-group"
```

Comment:
Any chance we could add a `tabIndex` here to take care of the focus order within the map? I think the `LayerSelector` should receive focus before the map attributions if possible!!

Reply:
I think the difficulty here is how we get the focus into the selector to begin with. I'll make an issue from this for anyone in the future who'd like to improve this!